src/bin/mmix-sim.rs

```diff
@@ -0,0 +1,50 @@
+extern crate mmix;
+
+use mmix::machine::Machine;
+
+use std::fs::File;
+use std::io::{Error, Read};
+
+fn main() {
```

Comment:
I think it is fine the way it is. Jonas suggested this method in the discussion under Issue #11